cmd/guacone/cmd/vulnerability.go

```diff
@@ -606,7 +606,7 @@ func searchPkgViaHasSBOM(ctx context.Context, gqlclient graphql.Client, searchSt
 	return path, tableRows, nil
 }
 
-func findDepPkgVersionIDs(ctx context.Context, gqlclient graphql.Client, depPkgType string, depPkgNameSpace string, depPkgName string, versionRange string) ([]string, error) {
+func FindDepPkgVersionIDs(ctx context.Context, gqlclient graphql.Client, depPkgType string, depPkgNameSpace string, depPkgName string, versionRange string) ([]string, error) {
```

Comment:
If this is copied out now, then it can be removed here and the one in pkg/dependencies can be called.

Reply:
Removed the duplicate one in vulnerability.go